pubtools/pulplib/_impl/model/repository/file.py

```diff
@@ -40,6 +40,12 @@ class FileRepository(Repository):
         type=bool,
         validator=validators.instance_of(bool),
     )
+    """True if this is a sigstore repository, used for container image manifest
```

Comment:
Can you please leave this one out?  The convention here is not to duplicate doc strings in the case where an attribute is being inherited from the base class.  This is why for example "type" and "mutable_urls" have no doc string.

Reply:
Sure, will update. Thanks!